gratipay/utils/fake_data.py

```diff
@@ -237,13 +281,24 @@ def populate_db(db, num_participants=100, num_tips=200, num_teams=5, num_transfe
         participants.append(fake_participant(db))
 
     print("Making Teams")
-    for i in xrange(num_teams):
-        team = fake_participant(db, number="plural")
-        #Add 1 to 3 members to the team
-        members = random.sample(participants, random.randint(1, 3))
-        for p in members:
-            team.add_member(p)
-        participants.append(team)
+    teams = []
+    teamowners = random.sample(participants, num_teams)
+    for teamowner in teamowners:
+        teams.append(fake_team(db, teamowner))
+
+    print("Making Subscriptions")
```

Comment:
Have we stopped making `tips` already in `fake_data`? (`tips` are what `subscriptions` replaced.) If not, let's do so on this PR.

[!m](http://motivate.im/) @rorepo